pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -174,7 +174,7 @@ let
       then ["--enable-multilib" "--disable-libquadmath"]
       else ["--disable-multilib"])
     ++ lib.optional (!enableShared) "--disable-shared"
-    ++ lib.singleton (lib.enableFeature enablePlugin "plugin")
+    ++ lib.singleton (lib.enableFeature enablePlugins "plugin")
```

Comment:
While `gcc/gcc` has only `--enable-plugin` `m4` code contains both flags:

```
$ git grep plugin |& fgrep AC_ARG
config/gcc-plugin.m4:   AC_ARG_ENABLE(plugin,
config/plugins.m4:  AC_ARG_ENABLE([plugins],
```

It is really scary that `enablePlugins` enable not the one you expect from the name.

I find `enablePlugin` and `enablePlugins` equally non-descriptive for what it does. But at least `enablePlugin` matches the configure option. There are at least 3 things I know of that `gcc` has to do with plugins:

- `liblto_plugin.so`: `binutils` linker plugin
- plugin API
- `libcc1plugin.so`

I suggest picking something more distinctive across the 3. Maybe `enablePluginApi`? Or not do the rename at all.

Reply:
Yikes, I didn't know that there was an `--enable-plugin` in there.